app/api/v2/models/orders_model.py

```diff
@@ -148,12 +149,16 @@ def cancelled_or_delivered(parcel_id):
         if order:
             return True
 
-    @classmethod
-    def get_by_specific_user(cls, user_id):
+    @staticmethod
+    def get_by_specific_user(user_id):
         """method for getting orders made by a specific user"""
-        try:
-            db.cursor.execute(
-                "SELECT * FROM parcels WHERE user_id = %s ", (user_id))
-            db.cursor.dictfetchall()
-        except Exception as e:
-            return {"Message": e}
+        db.cursor.execute("SELECT * FROM parcels WHERE user_id = %s", (user_id,))
+        order = db.cursor.fetchall()
+        return order
+
+    @staticmethod
+    def get_by_user(user_id):
```

Comment:
This method is a duplicate of the above method, `get_by_specific_user`. Any reason why you defined it?